sdks/wasm/Makefile

```diff
@@ -17,6 +17,7 @@ DRIVER_CONF=release
 EMSCRIPTEN_SDKDIR=$(TOP)/sdks/builds/toolchains/emsdk
 EMCC=source $(TOP)/sdks/builds/toolchains/emsdk/emsdk_env.sh && emcc
 WASM_BCL_DIR=$(TOP)/sdks/out/wasm-bcl/wasm
+WASM_TOOLS_DIR=$(TOP)/sdks/out/wasm-tools
```

Comment:
This clearly isn't going to work: this `$(TOP)/sdks/out/wasm-bcl/wasm` is pretty much the copy/pasting of `mcs/class/lib/wasm`, so simply "creating" a new `$(TOP)/sdks/out/wasm-tools` to which nothing is ever copied or generated is not good.

What is it exactly you are trying to achieve here?